### PR TITLE
Add toolchain mingw compilers

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,6 +45,28 @@ outputs:
...
+    run_exports:
+      strong:
+        - m2w64-gcc-libs-core
+
+  - name: toolchain_m2w64_cxx_{{ target_platform }}
+    requirements:
+      - m2w64-toolchain
+    run_exports:
+      strong:
+        - m2w64-gcc-libs-core
```
Comment (jakirkham) [inline recipe/meta.yaml]: Looks like `m2w64-gcc-libs` requires this and has `libstdc++`. So maybe we should be adding it here.

**Comment by jakirkham** [comment]
Not sure how we want to handle OpenMP, but `m2w64-gcc-libs` includes `libgomp`. So we should include it wherever we expect libraries using OpenMP to get it.

**Comment by mingwandroid** [comment]
Just add `m2w64-gcc-libs` anytime these compilers are used via strong run_exports so you no not need to worry about it.

The cost is minimal and for our main usage of these for R, `openmp` should be used always anyway.

**Comment by isuruf** [comment]
@mingwandroid, @jakirkham, can you check again?

**Comment by jakirkham** [comment]
LGTM

**Comment by isuruf** [comment]
I don;t understand the travis-ci failure

**Pull request merged**